java/ql/test/experimental/query-tests/security/CWE-555/CredentialsInPropertiesFile.ql

```diff
@@ -0,0 +1,24 @@
+/**
+ * @name Cleartext Credentials in Properties File
+ * @description Finds cleartext credentials in Java properties files.
+ * @kind problem
+ * @id java/credentials-in-properties
+ * @tags security
+ *       external/cwe/cwe-555
+ *       external/cwe/cwe-256
+ *       external/cwe/cwe-260
+ */
+
+/*
+ * Note this query requires properties files to be indexed before it can produce results.
+ * If creating your own database with the CodeQL CLI, you should run
+ * `codeql database index-files --language=properties ...`
+ * If using lgtm.com, you should add `properties_files: true` to the index block of your
```

Comment:
Usually a test query doesn't have metadata, block comments and so on -- this isn't a "real" query, but just a directive how to produce a result set for the test suite. Suggest removing all leading comments and replacing with `/* Note this is similar to `src/experimental/Security/CWE/CWE-555/CredentialsInPropertiesFile.ql` except we do not filter out test files. */`

Reply:
Updated